python/lzreposync/src/lzreposync/__init__.py

```diff
@@ -0,0 +1,218 @@
+        "--url",
+        "-u",
+        help="The target url of the remote repository of which we'll "
+        "parse the metadata",
+        dest="url",
+        type=str,
+        default=None,
+    )
+
+    parser.add_argument(
+        "-n",
+        "--name",
+        help="Name of the repository",
+        dest="name",
+        type=str,
+        default="noname",
+    )
+
+    parser.add_argument(
+        "-d",
+        "--debug",
+        help="Show debug messages",
+        action="store_const",
+        dest="loglevel",
+        const=logging.DEBUG,
+        default=logging.INFO,
+    )
+
+    parser.add_argument(
+        "-c",
+        "--cache",
+        help="Path to the cache directory",
+        dest="cache",
+        default=".",
+        type=str,
+    )
+
+    parser.add_argument(
+        "-b",
+        "--batch-size",
+        help="Size of the batch (num of packages by batch)",
+        dest="batch_size",
+        default=20,
+        type=int,
+    )
+
+    parser.add_argument(
+        "-a",
+        "--arch",
+        help="A filter for package architecture. Can be a regex, for example: 'x86_64',  '(x86_64|arch_64)'",
+        default=".*",
+        dest="arch",
+        type=str,
+    )
+
+    parser.add_argument(
+        "--channel",
+        help="The channel label of which you want to synchronize repositories",
+        dest="channel",
+        type=str,
+        default=None,
+    )
+
+    parser.add_argument(
+        "--type",
+        help="Repo type (yum or deb)",
+        dest="repo_type",
+        type=str,
+        default=None,
+    )
+
+    parser.add_argument(
+        "--import-updates",
+        help="Import related patches/updates",
+        action="store_true",
+        dest="import_updates",
+        default=False,
+    )
+
+    parser.add_argument(
+        "--create-channel",
+        help="Create a new channel by providing the 'channel_label', and the 'channel_arch' eg: x86_64.\n"
+        "Eg: --create-channel test_channel x86_64",
+        dest="channel_info",
+        type=str,
+        nargs=2,
+    )
+
+    args = parser.parse_args()
+
+    # Creating a new channel
+    if args.channel_info:
+        channel_label, channel_arch = args.channel_info[0], args.channel_info[1]
+        print(
+            f"Creating a new channel with label: {channel_label}, and arch: {channel_arch}"
+        )
+        try:
+            channel = create_channel(
+                channel_label=channel_label, channel_arch=channel_arch
+            )
+            print(
+                f"Info: successfully created channel: {channel_label} -> id={channel.get_id()}, name={channel.get_label()}"
+            )
+        except ChannelAlreadyExistsException:
+            print(f"Warn: failed to create channel {channel_label}. Already exists !!")
+        return
+
+    arch = args.arch
+    if arch != ".*":
+        # pylint: disable-next=consider-using-f-string
+        arch = "(noarch|{})".format(args.arch)
```

Comment:
Instead of silencing pylint, we should follow the rules we have set it for it. In this case it's as simple as:
```suggestion
        arch = f"(noarch|{args.arch})"
```